tests/linear_model/test_glm.py

```diff
@@ -203,3 +220,79 @@ def test_logistic_predict_proba_shape():
     lr.fit(X, y)
     prob = lr.predict_proba(X)
     assert prob.shape == (100, 2)
+
+@pytest.mark.parametrize(
+    "reg_type,data_generator", 
+    [(LinearRegression, "single_chunk_regression"), 
+    (LogisticRegression, "single_chunk_classification"),
+    (PoissonRegression, "single_chunk_count_classification")
+    ]
+)
+def test_model_coef_dask_numpy(reg_type, data_generator, request):
+    """Tests that models return same coefficients and intercepts with array types"""
+    X, y = request.getfixturevalue(data_generator)
+    np_mod, da_mod = reg_type(fit_intercept=True), reg_type(fit_intercept=True)
+    da_mod.fit(X, y)
+    np_mod.fit(X.compute(), y.compute())
+    np.testing.assert_allclose(da_mod.coef_, np_mod.coef_)
+    np.testing.assert_allclose(da_mod.intercept_, np_mod.intercept_)
+
+@pytest.mark.parametrize(
+    "reg_type, skl_reg_type, skl_params, data_generator", 
+    [(LinearRegression, sklLinearRegression, {}, "single_chunk_regression"), 
+    (LogisticRegression, sklLogisticRegression, {'penalty' : 'none'}, "single_chunk_classification"),
+    (PoissonRegression, sklPoissonRegressor, {'alpha' : 0}, "single_chunk_count_classification")
+    ]
+)
+def test_model_against_sklearn(reg_type, skl_reg_type, skl_params, data_generator, request):
+    """
+    Tests accuracy of model predictions
+
+    Notes:
+        1. Dask ML 'newton' solver does not implement a penalty, only
+            set arbitarily high for clarity of non-regularization.
+        2. This method only works for `fit_intercept=False`
```

Comment:
Why? `fit_intercept=True` below.

Reply:
This has been removed and updated in the test code.